### PR TITLE
Add framebufferScaleFactor to XRWebGLLayerInit

### PR DESCRIPTION
**Pull request by klausw**

### CODE REVIEW

**Comment by klausw** [comment]
FYI, as discussed with @elalish , this isn't necessarily intended for merging as-is, but I wanted to upload it in case it's useful as reference for future work.

File: packages/model-viewer/src/three-components/ARRenderer.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export class ARRenderer extends EventDispatcher {
     await gl.makeXRCompatible();
 
     session.updateRenderState(
-        {baseLayer: new XRWebGLLayer(session, gl, {alpha: true})});
+        {baseLayer: new XRWebGLLayer(session, gl, {alpha: true, framebufferScaleFactor: 0.5})});
```
Comment (elalish) [inline packages/model-viewer/src/three-components/ARRenderer.ts:98]: Thanks @klausw I'll definitely be making use of this in the future. Meanwhile, would you mind removing this and pulling master so I can merge the types?

Comment (klausw) [reply packages/model-viewer/src/three-components/ARRenderer.ts]: Done.

**Comment by klausw** [comment]
There's a unit test failure, but I don't think this one is related to my
change:

@google/3dom: 09 04 2020 23:27:07.108:ERROR [config]: Error in config file!
22 <https://github.com/klausw/model-viewer/runs/575507817#step:10:22>@google/3dom:
Error: BROWSER_STACK_USERNAME and BROWSER_STACK_ACCESS_KEY must be set with
USE_BROWSER_STACK

**Pull request merged**